src/main/kotlin/com/zepben/evolve/database/sqlite/metrics/MetricsDataSourceWriter.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.metrics
```

Comment:
todo: This is a non SQLite database class that is in the sqlite package. Please move it to an appropriate package.

Reply:
discussion: @Lax125 @gcarwardine Upon reflection, this is using the SQLite tables, so it is related to SQLite, and as such will only work for databases that share syntax.

Do we want to keep it in the `sqlite` package as an indicator that it is linked to SQLite syntax?